CMakeLists.txt

```diff
@@ -197,7 +197,7 @@ endif()
 ###############################################################################
 
 if(EPROSIMA_BUILD AND NOT EPROSIMA_INSTALLER_MINION)
-    set(INTERNAL_DEBUG ON)
+    option(INTERNAL_DEBUG ON)
```

Comment:
It is not a duplicity. The second one set it to OFF in case it has not been set yet.
However, I also think it would be better to unify both in the same CMakeLists.